test/action.spec.js

```diff
@@ -0,0 +1,112 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const mock = require('mock-require');
+const log = require('npmlog');
+const path = require('path');
+
+const action = (config, args) => require('../src/action')(config, args || []);
+
+log.level = 'silent';
+
+describe('action', () => {
+
+  beforeEach(() => {
+    delete require.cache[require.resolve('../src/action')];
```

Comment:
I'm not saying that there are any cons here. I have some reasons to do it in a proposed way: 
- It makes tests simpler to read and understand. In my opinion mocking something is more obvious than manual require cache cleaning
- In a case of consistency; we're using mock-require library anyway, so I'd like to stick to one approach mocking require calls

I'm fine to merge it as it is, but then I'll compose a PR to change it to mock-require. Deal?